classes/module/Module.php

```diff
@@ -3144,6 +3144,23 @@ protected function trans($id, array $parameters = array(), $domain = null, $loca
         $parameters['legacy'] = 'htmlspecialchars';
         return $this->getTranslator()->trans($id, $parameters, $domain, $locale);
     }
+
+    /**
+     * Check if the module uses the new translation system
+     * @return bool
+     */
+    public function isUsingNewTranslationsSystem()
+    {
+        $moduleName = $this->name;
+        $domains = array_keys($this->context->getTranslator()->getCatalogue()->all());
+        $moduleName = preg_replace('/^ps_(\w+)/', '$1', $moduleName);
+
+        if (count(preg_grep('/'.$moduleName.'/i', $domains))) {
```

Comment:
Hello @eternoendless 
What do you think of that :
`if (stripos('#' . implode($domains), $moduleName)) {
            return true;
        }`